src/modules/model/views/Add.vue

```diff
@@ -0,0 +1,71 @@
+<script setup>
+import { useStore } from "@/composables/useStore";
+import { onMounted, ref, useModel } from "vue";
+import { db } from "../../../firebase.config";
+import Alert from "../../../components/alert.vue";
+import Body from "../../../components/body.vue";
+
+const { content } = useStore();
+
+const newModel = ref("");
+const selectedMark = ref("");
+
+async function handleCreateModel() {
+  const res = await content.model.addModel({
+    descModel: newModel.value,
+    statusModel: true,
+    idMark: selectedMark.value,
+  });
+  if (res) {
+    alert("Modelo criado com sucesso");
+    window.location.href = "/model";
```

Comment:
o ideal aqui é usar o router com a função push
```suggestion
    window.location.href = "/model";
```

Reply:
alterado para this.$router.push("/model");